book/src/configuration.md

```diff
@@ -45,6 +45,7 @@ hidden = false
 | `auto-info` | Whether to display infoboxes | `true` |
 | `true-color` | Set to `true` to override automatic detection of terminal truecolor support in the event of a false negative. | `false` |
 | `rulers` | List of column positions at which to display the rulers. Can be overridden by language specific `rulers` in `languages.toml` file. | `[]` |
+| `bufferline` | Renders a line at the top of the editor displaying open buffers. Can be `always`, `never` or `multiple` (only shown if more than one buffer is in use) | `multiple` |
```

Comment:
I'd prefer this default to `never`. I think it falls under the category of features that some really like and others find distracting or overly verbose like visible whitespace (also indent guides for example).

Reply:
There was some feedback earlier that `multiple` would be a good default, but if there is agreement I am happy to change it back to never.

Reply:
I would personally vote for having `multiple` by default since it's more discoverable that way and can be easily disabled in the config. But then again that argument could be extended to every single new feature so :)

Reply:
I am personally also of the opinion that `multiple` is the better default option. Anyone who cares that much about the wasted space of a single line will probably have a bunch of custom options set in their config anyways, and adding one more to hide it should be trivial.

Reply:
I'm curious what @archseer thinks about the default behavior.

I was going to say that the default keymap favors the buffer picker for switching buffers but that's not actually true: `gn`/`gp` are the same number of keystrokes. I'm definitely biased as I only ever use the buffer picker.

> Anyone who cares that much about the wasted space of a single line will probably have a bunch of custom options set in their config anyways, and adding one more to hide it should be trivial.

It's more consistent with the current configuration options to make the reverse argument: anyone who wants to enable a cosmetic feature will probably have a bunch of custom options set in their config anyways, and adding one more to show it should be trivial. Examples: visible whitespace, indent guides, cursorline, rulers, custom cursor shapes.

> it's more discoverable [enabled by default]

Hmm yeah I agree. IMO though a toggle mode like #2849 would be the ideal approach to that - enabling new features by default as a general policy might end up with a busy/overwhelming default experience.

Reply:
I'll go with Never by default, based on @archseer's comment below